components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this from the Photon icon pack?
* https://github.com/FirefoxUX/photon-icons/tree/master/icons/android
* https://design.firefox.com/icons/viewer/

Reply:
Yes it is.